ChangeLog.d/fix-rsa-pss-rsae-compitable-issue.txt

```diff
@@ -0,0 +1,7 @@
+Bugfix
```

Comment:
This doesn't describe to an end user what has been done here. For starters it is referring to internal code, and for seconds this describes the situation prior to this fix.
Would something more like "Add RSA PSA RSAE support for TLS1.2 client" be more descriptive?
Also, is this a bugfix or a new feature?